### PR TITLE
Feature/admin dashboard

### PR DESCRIPTION
**Pull request by DanteTheDevil**

### CODE REVIEW

File: src/app/admin-panel/dashboard/dashboard.component.ts [hunk 1]
```diff
@@ -0,0 +1,164 @@
+import {Component, OnInit, ViewChild} from '@angular/core';
```
Comment (dmitryy90) [inline src/app/admin-panel/dashboard/dashboard.component.ts]: indentation

File: src/app/admin-panel/dashboard/dashboard.component.ts [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+  constructor(private subjectService: SubjectService, private teacherService: TeacherService,
+              private classService: ClassService, private studentService: StudentsService) { }
+
+  /**
+   * Method returns values of dashboard data
+   * @returns - Array with values of dashboard data
+   */
+  public dataValues(): number[] {
+    return Object.values(this.data);
+}
```
Comment (dmitryy90) [inline src/app/admin-panel/dashboard/dashboard.component.ts]: indentation

File: src/app/models/student.ts [hunk 3]
```diff
@@ -1,6 +1,6 @@
 export interface Student {
     avatar?: string;
-    classId: number
+    classId: number;
     classe?: string;
```
Comment (dmitryy90) [inline src/app/models/student.ts]: classe typo?

Comment (DanteTheDevil) [reply src/app/models/student.ts]: Sorry, i dont know who is the owner of this interface, it's not mine

File: src/app/admin-panel/dashboard/dashboard.component.ts (hunk 2 above)
Comment (DanteTheDevil) [reply src/app/admin-panel/dashboard/dashboard.component.ts]: Fixed

File: src/app/admin-panel/dashboard/dashboard.component.ts (hunk 1 above)
Comment (DanteTheDevil) [reply src/app/admin-panel/dashboard/dashboard.component.ts]: Fixed

File: src/app/models/student.ts (hunk 3 above)
Comment (dmitryy90) [reply src/app/models/student.ts]: ask the owner to fix this

Comment (dmitryy90) [reply src/app/models/student.ts]: or step into a trail of goodness =)

**Pull request merged**